### PR TITLE
Add Tipe Logo

### PR DESCRIPTION
**Pull request by dtarellano**

### CODE REVIEW

File: src/components/Tipe/Tipe.jsx [hunk 1]
```diff
@@ -4,13 +4,11 @@ export default () => {
...
-        alt="Tipe Logo"
-        style={{
-          width: '50%',
-          height: '50%'
-        }}
-      />
+        src="https://cdn.tipe.io/tipe/tipe-ad.png?h=50" srcSet="https://cdn.tipe.io/tipe/tipe-ad.png?h=50 1x,
+          https://cdn.tipe.io/tipe/tipe-ad.png?h=50&dpr=2 2x,
+          https://cdn.tipe.io/tipe/tipe-ad.png?h=50&dpr=3 3x"
+          alt="Tipe" width="86" height="50"
```
Comment (gdi2290) [inline src/components/Tipe/Tipe.jsx]: the image also needs the width 86

**Comment by dtarellano** [comment]
screen shot 2018-05-29 at 5 08 22 pm

File: src/components/Tipe/Tipe.jsx (hunk 1 above)
Comment (dtarellano) [reply src/components/Tipe/Tipe.jsx]: done

**Comment by dtarellano** [comment]
screen shot 2018-05-29 at 5 36 21 pm

**Comment by dtarellano** [comment]
screen shot 2018-05-29 at 6 44 00 pm

**Comment by dtarellano** [comment]
styling now looks correct on all testable devices

**Pull request merged**